<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>311</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionClear_All"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <widget class="QMenu" name="menuBullets">
     <property name="title">
      <string>Bullets</string>
     </property>
     <addaction name="actionDisc"/>
     <addaction name="separator"/>
     <addaction name="actionCircle"/>
     <addaction name="actionSqure"/>
    </widget>
    <widget class="QMenu" name="menuNumbering">
     <property name="title">
      <string>Numbering</string>
     </property>
     <addaction name="actionArabic"/>
     <addaction name="actionUppercase_Normal"/>
     <addaction name="actionLowercase_Roman"/>
    </widget>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="menuBullets"/>
    <addaction name="menuNumbering"/>
    <addaction name="actionColor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/images.png</normaloff>:/imgs/resources/images.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/download (2).png</normaloff>:/imgs/resources/download (2).png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/images (1).png</normaloff>:/imgs/resources/images (1).png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/download.jpg</normaloff>
     <normalon>:/imgs/resources/download.jpg</normalon>:/imgs/resources/download.jpg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>resources/copy q.png</normaloff>
     <normalon>:/imgs/resources/copy q.png</normalon>resources/copy q.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/download (4).png</normaloff>:/imgs/resources/download (4).png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/images (2).png</normaloff>:/imgs/resources/images (2).png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/redo.png</normaloff>:/imgs/resources/redo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/download (3).jpg</normaloff>:/imgs/resources/download (3).jpg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/images.png</normaloff>
     <normalon>:/imgs/resources/images.png</normalon>:/imgs/resources/images.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/Zoomin.png</normaloff>:/imgs/resources/Zoomin.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/zoomout.jpg</normaloff>:/imgs/resources/zoomout.jpg</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset>
     <normaloff>resources/selectAll.png</normaloff>
     <normalon>:/imgs/resources/selectAll.png</normalon>resources/selectAll.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionClear_All">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/clearAll2.png</normaloff>:/imgs/resources/clearAll2.png</iconset>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/bold.png</normaloff>:/imgs/resources/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/italic.png</normaloff>:/imgs/resources/italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/underline.png</normaloff>:/imgs/resources/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionDisc">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/disc.png</normaloff>:/imgs/resources/disc.png</iconset>
   </property>
   <property name="text">
    <string>Disc</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/imgs/resources/circle.png</normaloff>:/imgs/resources/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionSqure">
   <property name="icon">
    <iconset>
     <normalon>:/imgs/resources/square.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Squre</string>
   </property>
  </action>
  <action name="actionArabic">
   <property name="icon">
    <iconset>
     <normaloff>resources/arabicNum.png</normaloff>
     <normalon>:/imgs/resources/arabicNum.png</normalon>resources/arabicNum.png</iconset>
   </property>
   <property name="text">
    <string>Arabic</string>
   </property>
  </action>
  <action name="actionUppercase_Normal">
   <property name="icon">
    <iconset>
     <normaloff>resources/Uppercaseroman.png</normaloff>
     <normalon>:/imgs/resources/Uppercaseroman.png</normalon>resources/Uppercaseroman.png</iconset>
   </property>
   <property name="text">
    <string>Uppercase Roman</string>
   </property>
  </action>
  <action name="actionLowercase_Roman">
   <property name="icon">
    <iconset>
     <normaloff>resources/Uppercaseroman.png</normaloff>
     <normalon>:/imgs/resources/Uppercaseroman.png</normalon>resources/Uppercaseroman.png</iconset>
   </property>
   <property name="text">
    <string>Lowercase Roman</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset>
     <normalon>:/imgs/resources/color.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
